core/internal/cltest/cltest.go

```diff
@@ -323,20 +329,18 @@ func NewApplicationWithKey(t *testing.T, flagsAndDeps ...interface{}) (*TestAppl
 func NewApplicationWithConfigAndKey(t testing.TB, c *configtest.TestEVMConfig, flagsAndDeps ...interface{}) (*TestApplication, func()) {
 	t.Helper()
 
-	app, cleanup := NewApplicationWithConfig(t, c, flagsAndDeps...)
+	app, cleanup := NewApplicationWithConfig(t, tc, flagsAndDeps...)
+	require.NoError(t, app.KeyStore.Unlock(Password))
 	for _, dep := range flagsAndDeps {
 		switch v := dep.(type) {
-		case ethkey.Key:
-			MustAddKeyToKeystore(t, &v, app.KeyStore.Eth())
+		case ethkey.KeyV2:
```

Comment:
What is the difference between ethkey.Key and ethkey.KeyV2?

Reply:
ethkey.Key is the "old key" we only need it as a model for the eth_keys table. ethkey.KeyV2 is the new key. Nearly identical API, but a few added features to make it compatible with the new keystore.

Reply:
Is the `eth_keys` table not removed as part of this PR?

Reply:
[see discussion here](https://github.com/smartcontractkit/chainlink/pull/4812#discussion_r694445689)